frontend/src/pages/UserPage/routes.ts

```diff
@@ -40,6 +41,11 @@ export const routes: RoutesGroup[] = [
                 name: 'Offered Services',
                 path: 'offered-services',
                 component: OfferedServices
+            },
+            {
+                name: 'Pending service requests',
+                path: 'service-requests/pending',
+                component: PendingServiceRequests
             }
         ]
     },
```

Comment:
It is best to add a fallback route with component named PageNotFound or some similar name.